### PR TITLE
More cython lint checks

### PR DESCRIPTION
**Pull request by fchapoton**
fixing a few more, activating them

### CODE REVIEW

File: src/flint/types/fq_default_poly.pyx [hunk 1]
```diff
@@ -1324,7 +1324,7 @@ cdef class fq_default_poly(flint_poly):
         """
         G, S, _ = self.xgcd(other)
         if not G.is_one():
-            raise ValueError(f"polynomial has no inverse modulo {other = }")
+            raise ValueError(f"polynomial has no inverse modulo {other=}")
```
Comment (oscarbenjamin) [inline src/flint/types/fq_default_poly.pyx:1327]: It looks like this changes the output:
```python
In [7]: x = fmpz_poly([0, 1])

In [8]: p = 1 + x**2 + x

In [9]: f'{p=}'
Out[9]: 'p=x^2 + x + 1'

In [10]: f'{p = }'
Out[10]: 'p = x^2 + x + 1'
```
Does cython-lint complain about this?

Comment (fchapoton) [reply src/flint/types/fq_default_poly.pyx:1327]: ah, damn. Yes, it seems that cython-lint is not smart enough..

**Comment by fchapoton** [comment]
undone now

**Comment by oscarbenjamin** [comment]
Looks good. Thanks!

**Pull request merged**